manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: KhronosGroup.VulkanSDK
+PackageVersion: 1.3.268.0
+PackageLocale: en-US
+Publisher: LunarG Inc.
+PublisherUrl: https://www.lunarg.com/
+PublisherSupportUrl: https://vulkan.lunarg.com/issue/home
+PrivacyUrl: https://vulkan.lunarg.com/sdk/home#content/popup/privacy/lg
+PackageName: Vulkan SDK
+PackageUrl: https://vulkan.lunarg.com/sdk/home
+License: MIT
+LicenseUrl: https://vulkan.lunarg.com/software/license/vulkan-1.3.268.0-windows-license-summary.txt
+Copyright: Copyright (c) 2015-2023 LunarG, Inc.
+CopyrightUrl: https://vulkan.lunarg.com/software/license/vulkan-1.3.268.0-windows-license-summary.txt
+ShortDescription: The Vulkan SDK provides Vulkan application developers with essential tools to accelerate the development process.
+Moniker: vulkansdk
+Tags:
+- vulkan-api
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0
+Agreements:
+  - AgreementLabel: LunarG Inc. Vulkan SDK License Agreement
+    Agreement: |
+      Copyright 2016-2023 LunarG Inc.
+
+      The Vulkan SDK is comprised of 100% open-source components. The majority of the materials are MIT or Apache 2.0 licenses. The Vulkan SDK licensing registry (found at vulkan.lunarg.com) discloses all components in the SDK and their corresponding open source license. The SDK itself is not licensed because to do so would re-attribute licenses to all the things included in the SDK.
+
+      ALL INFORMATION HERE IS PROVIDED "AS IS." LUNARG MAKES NO REPRESENTATIONS OR WARRANTIES, EXPRESS OR IMPLIED, WITH REGARD TO THIS LIST OR ITS ACCURACY OR COMPLETENESS, OR WITH RESPECT TO ANY RESULTS TO BE OBTAINED FROM USE OR DISTRIBUTION OF THE LIST. BY USING OR DISTRIBUTING THIS LIST, YOU AGREE THAT IN NO EVENT SHALL LUNARG BE HELD LIABLE FOR ANY DAMAGES WHATSOEVER RESULTING FROM ANY USE OR DISTRIBUTION OF THIS LIST, INCLUDING, WITHOUT LIMITATION, ANY SPECIAL, CONSEQUENTIAL, INCIDENTAL OR OTHER DIRECT OR INDIRECT DAMAGES.
```

Comment:
```suggestion
Agreements:
  - AgreementUrl: https://vulkan.lunarg.com/software/license/vulkan-1.3.268.0-windows-license-summary.txt
```

Reply:
That link does not include all that is agreed upon on listed. The "AS IS" warning for instance is not part in the URL you link to so I'm not sure if this would legally be the same agreement - though the link does seem to be way more elaborate but the warning that it might miss stuff is no longer part of that...